presto-main/src/main/java/com/facebook/presto/cost/ComparisonStatsCalculator.java

```diff
@@ -33,158 +33,159 @@
 {
     private ComparisonStatsCalculator() {}
 
-    public static Optional<PlanNodeStatsEstimate> comparisonExpressionToLiteralStats(
+    public static PlanNodeStatsEstimate estimateExpressionToLiteralComparison(
             PlanNodeStatsEstimate inputStatistics,
-            Optional<Symbol> symbol,
-            SymbolStatsEstimate expressionStats,
-            OptionalDouble doubleLiteral,
-            ComparisonExpression.Operator operator)
+            SymbolStatsEstimate expressionStatistics,
+            Optional<Symbol> expressionSymbol,
+            OptionalDouble literalValue,
+            Operator comparisonOperator)
```

Comment:
please extract refactors to separate commits

Reply:
@kokosing 

> why not just operator? We are already in Comparison* class

I can rename it to `operator`, but than i think it makes sense to import class as `ComparisonExpression.Operator`. We got a lot of `Operator` classes through the codebase. What do you think?

> I think operator should go just after inputStatistics like we do in other places (like ArithmeticBinaryExpression)

Could you please send me the code reference?